decidim/decidim
# Fix superposition in admin's error forms

#### :tophat: What? Why?

While reviewing #9786 I saw that we've broken the error forms on admin panel. This is probably related with the Redesign process, and as it's CSS I was hesitant to make this PR. 

My reasoning is that this only applies to the admin panel and for troubleshooting we need to see the error messages. Also, as the admin panel redesign will not be ready for v0.28 (we would only have the public interface redesign for that version) I think that we need to tackle these kinds of issues on admin UX. 

Feel free to correct me or close this PR as wontfix if you don't see it clear. 

#### Testing

1. Sign in as admin
2. Go to admin panel 
3. Generate a validation error in a form.

For instance:
1. Go to a step of a process
2. Remove the title
3. Put the end date before the start date
4. Save the form
5. See the error

### :camera: Screenshots

#### Before

![Selection_319](https://user-images.githubusercontent.com/717367/193011130-80f4637a-25ab-4b9e-94ca-edb03c606d6a.png)


#### After
![Selection_320](https://user-images.githubusercontent.com/717367/193011114-dfa945e6-822c-43f8-9653-0361f64fdb02.png)

Mind that there's more job to be made here regarding the margins. We could tackle that on this same PR, another PR, or during the admin Redesign. Whatever strategy you prefer. 

:hearts: Thank you!
